app/views/protocol_importers/templates/_note.html.erb

```diff
@@ -1,4 +1,6 @@
 <p class="step-description-component-note"><b><%= t('protocol_importers.templates.note.title') %></b><br/>
-  <%= "#{t('protocol_importers.templates.note.author')}: #{item[:author]}" %><br/>
+  <% if item[:name] %>
+    <%= "#{t('protocol_importers.templates.note.author')}: #{item[:author]}" %><br/>
+  <% end %>
   <%= item[:body] %></p>
```

Comment:
Should you the same check for `:body`? It appears it can be empty too https://apidoc.protocols.io/#comment-object

Reply:
We do not have `Body: ` label there, so won't print anything if `:body` is not present. But I add a check. Let's be consistent.